Makefile

```diff
@@ -71,13 +71,13 @@ fixcs: docker-start bin/tools/cs-fixer/vendor ### Run PHP CS-Fixer
 	@${DOCKER_PHP} bin/tools/cs-fixer/vendor/friendsofphp/php-cs-fixer/php-cs-fixer --no-interaction --diff -v fix
 
 bin/tools/cs-fixer/vendor: vendor bin/tools/cs-fixer/composer.json bin/tools/cs-fixer/composer.lock
-	@${DOCKER_PHP} composer bin cs-fixer update
+	@${DOCKER_PHP} composer bin cs-fixer install
```

Comment:
I had to change this, otherwise psalm and csfixer's composer.lock will always be updated